app/main.py

```diff
@@ -1 +1,20 @@
-# write your code here
+from app.players.elves.elf_ranger import ElfRanger
+from app.players.elves.druid import Druid
+from app.players.dwarves.dwarf_warrior import DwarfWarrior
+from app.players.dwarves.dwarf_blacksmith import DwarfBlacksmith
+
+
+def calculate_team_total_rating(
+        players: list[ElfRanger, Druid, DwarfWarrior, DwarfBlacksmith]
```

Comment:
Better indicate that you expect to obtain here any instances of classes that inherit from `Player` class instead of listing all children of this class